<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbstractGraphicsView</class>
 <widget class="QWidget" name="AbstractGraphicsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>298</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="UTGraphicsView" name="view">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="rightRuler" native="true">
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="bottomRuler" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="leftRuler" native="true">
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="topRuler" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UTGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>Waveform/UTGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
